### ISSUE TITLE
Search metadata on gallery page

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#448]
Feature request (cos I cannot fidn what I am looking for):

On the galleries, could there be a search box to search the metatdata surrounding a gallery, album or image? I cannot find where I uploaded a bunch of images to apart from clicking through every gallery. Long, tedious and not good UX.

**Comment by pmackay** [comment]
What about if the site search indexed all that content? Its too complex to implement search boxes for individual pages.

**Comment by GilesGibson** [comment]
The problem comes with searching for a generic "word" that may be pertinent to a photo but also comes up in loads of forum entries and just swamps the results page with erroneous results, the user only wanted images. The flip side is that there will be times when someone does want to search the entire site. Is it possible to have a limiter in some way that the user could invoke? Just images, or just Forum or just Groups?

**Comment by pmackay** [comment]
Yes you can do a search and then filter by content type on the results page.

**Comment by GilesGibson** [comment]
could we not therefore have the filter as part of the search? Better UX.
